src/lib/providers/gst/gst.ts

```diff
@@ -112,4 +114,55 @@ export class Angulartics2GoogleGlobalSiteTag {
       }
     });
   }
+
+  /**
+   * User Timings Event in GST.
+   * @name userTimings
+   *
+   * @param properties Comprised of the mandatory fields:
+   *  - name (string)
+   *  - value (number - integer)
+   * Properties can also have the optional fields:
+   *  - category (string)
+   *  - label (string)
+   *
+   * @link https://developers.google.com/analytics/devguides/collection/gtagjs/user-timings
+   */
+  userTimings(properties: UserTimingsGst) {
+    if (
+      !properties ||
+      !properties.name ||
+      !properties.value
+    ) {
+      console.error(
+        'Properties name, and value are required to be set.',
```

Comment:
the typescript types already validate these exist right?

I think we can exclude the error checking

Reply:
That's partially true because Typescript is compile-time only so at runtime you will lose all checks (or if somebody doesn't use Typescript compiler at all). 
However, it is an Angular library, which has Typescript as the first-class citizen, so we can expect that TS will always be used. 

I've removed the checks and left only undefined assert, otherwise, it could throw nasty `Uncaught TypeError`.